ee/packages/abac/src/index.spec.ts

```diff
@@ -37,12 +38,34 @@ jest.mock('@rocket.chat/models', () => ({
 		removeById: (...args: any[]) => mockAbacDeleteOne(...args),
 		find: (...args: any[]) => mockAbacFind(...args),
 	},
+	Users: {
+		find: (...args: any[]) => mockUsersFind(...args),
+	},
 }));
 
-// Minimal mock for ServiceClass (we don't need its real behavior in unit scope)
-jest.mock('@rocket.chat/core-services', () => ({
-	ServiceClass: class {},
-}));
+// Minimal mock for ServiceClass plus MeteorError and Room used by the service
+jest.mock('@rocket.chat/core-services', () => {
+	class MeteorError extends Error {
+		public isClientSafe = true;
+
+		public readonly errorType = 'Meteor.Error';
+
+		public constructor(
+			public readonly error: string | number,
+			public readonly reason?: string,
+			public readonly details?: any,
+		) {
+			super(`${reason ? `${reason} ` : ''}[${String(error)}]`);
+		}
+	}
```

Comment:
Couldn't get it via `jest.requireActual('@rocket.chat/core-services').MeteorError`?

Reply:
I didn't know this existed, thx 🙏🏽